src/Application/Products/Commands/AddProduct/AddProductCommandHandler.cs

```diff
@@ -12,16 +12,19 @@ internal class AddProductCommandHandler : IRequestHandler<AddProductCommand, Pro
 {
     private readonly ILogger _logger;
     private readonly IProductRepository _productRepository;
+    private readonly IDateTime _dateService;
     private readonly IStockClient _stockClient;
     private readonly IValidator<Product> _validator;
 
     public AddProductCommandHandler(
         IProductRepository productRepository,
+        IDateTime dateService,
```

Comment:
Either it should be `IDateTimeService`, or the variable should be `dateTime`...
I guess the first sounds more kind of 'natural' (even though I don't like the 'service' word) :-)